src/App.tsx

```diff
@@ -1,35 +1,55 @@
 import React, { useEffect, useState } from 'react';
-import Feilside from './komponenter/Feilside';
-import NavFrontendSpinner from 'nav-frontend-spinner';
-import Søknad from './komponenter/Søknad';
-import { Panel } from 'nav-frontend-paneler';
+import Banner from './components/Banner';
+import Feilside from './components/feilside/Feilside';
 import hentToggles from './toggles/api';
-import { ToggleName, Toggles } from './typer/toggles';
+import NavFrontendSpinner from 'nav-frontend-spinner';
+import Språkvelger from './components/språkvelger/Språkvelger';
+import Søknad from './søknad/Søknad';
+import { ToggleName, Toggles } from './models/toggles';
+import { hentPersonData } from './utils/søknad';
+import useSøknadContext from './context/SøknadContext';
+import { PersonActionTypes, usePersonContext } from './context/PersonContext';
 
 const App = () => {
   const [toggles, settToggles] = useState<Toggles>({});
   const [fetching, settFetching] = useState<boolean>(true);
-
   const [error, settError] = useState<boolean>(false);
+  const { person, settPerson } = usePersonContext();
+  const { søknad, settSøknad } = useSøknadContext();
 
   useEffect(() => {
     const fetchData = () => {
       hentToggles(settToggles).catch((err: Error) => {
-        settError(true);
+        // settError(true);
```

Comment:
kommentert ut for å unngå feilsiden. Må tenke litt mer på hva vi skl gjøre med feilsiden og når den skal vises.